README.md

```diff
@@ -45,13 +59,37 @@ yarn scripts:addTrip
 
 You can run a migration service to add aggregation data on your timeseries. This is necessary because the trips documents can be huge and negatively impact the app performances. Therefore, we rely on an aggregated trip view on the app side. If the aggregation is missing, the trip won't be displayed.
 
+⚠️ Make sure to update your development URL in `./konnector-dev-config.json` in the field `"COZY_URL"` if you do not wish to use the default URL `http://cozy.localhost:8080`.
+
+⚠️ Your instance database must contain in its `io.cozy.accounts` the following document as-is in order to be able to find the `geoseries` collections, please insert it manually or with a mango request into the associated database:
+
+```JSON
+{
+  "_id": "23f8af047dfc736c8b841f22eb00464f",
+  "account_type": "tracemob",
+  "auth": {
+    "credentials_encrypted": "sRDK6wGI9iAXo9P4bbMNjC6SA=",
+    "login": "test",
+    "providerId": "0"
+  },
+  "data": {
+    "lastSavedManualDate": "2021-11-17T09:23:51.280Z",
+    "lastSavedTripDate": "2021-12-01T16:23:15.179489-08:00"
+  },
+  "identifier": "login",
+  "state": null
+}
+```
+
 ⚠️ You need to build the application before using services, to generate them.
 
 ```sh
 $ yarn build
 $ yarn service:timeseriesWithoutAggregateMigration
 ```
 
+⚠️ If the service is not working due to an invalid JWT token, please delete it from your filesystem and restart the service, it was probably created for another instance that the one you're querying for.
```

Comment:
It would be worth saying how to delete it, i.e. where it is stored.
Also, it seems relevant for every service, maybe this part could be moved in https://github.com/cozy/cozy.github.io/blob/dev/src/howTos/dev/services.md ?

Reply:
I think the path will depend on the user's machine. It is printed in the error message. I should add this to my message then. For the other part, I'm not sure what to say.

Reply:
updated